File: March 31, 2026.py
def alarm_check(alarm_time, wake_time):
    [hour_alarm, minute_alarm] = alarm_time.split(":")
    [hour_wake, minute_wake] = wake_time.split(":")
    time_alarm = int(hour_alarm) * 60 + int(minute_alarm)
    time_wakeup = int(hour_wake) * 60 + int(minute_wake)

    if time_wakeup - time_alarm > 10:
        return "late"
    elif time_wakeup - time_alarm >= 0:
        return "on time"

    return "early"

File: March 31, 2026.js
function alarmCheck(alarmTime, wakeTime) {

  const [hourAlarm, minuteAlarm] = alarmTime.split(":").map(Number);
  const [hourWake, minuteWake] = wakeTime.split(":").map(Number);
  const timeAlarm = (hourAlarm * 60) + minuteAlarm;
  const timeWakeUp = (hourWake * 60) + minuteWake;

  if (timeWakeUp - timeAlarm > 10) {
    return "late"
  } else if (timeWakeUp - timeAlarm >= 0) {
    return "on time"
  }
  
  return "early"
}